README.md

```diff
@@ -1,63 +1,87 @@
+# Overview
 
-The code in this repository will take away 95% of the development work required in implementing Universal Variable on your Shopify store.
+When implementing Qubit, you need to do two things from a technical perspective:
 
-#Overview
+1. Add the Smartserve container to the `<head>` of every page
+2. Emit emits based on user browsing behaviour
 
-When implementing [Universal Variable](http://tools.qubitproducts.com/uv) and [Opentag](http://www.qubitproducts.com/tag-management), you need to do two things from a technical perspective:
-
-1. Add the Opentag container to the `<head>` of every page
-2. Add the Universal Variable (UV) directly before the Opentag container on every page.
-
-The code in this repository handles both of these for you, using Shopify templates.
+The code in this repository handles both of these for you via using Shopify templates. We also bundle the UV API, which sets up an API that acts as a conduit for events. Any script can emit events using the API, even before smartserve.js loads and those events can be read and subscribed to by any script using the API.
 
 __Note__: This repository is not a Shopify App as such, it provides a series of sample logic that can be copied across to your [Shopify templates](http://docs.shopify.com/themes).
 
 
-#Installation
+# Installation
 
-##Opentag container
+## Smartserve Script
 
-Update the Opentag container at the bottom of `uv.liquid` and `uv-transaction.liquid` with your own unique id.
+Update the Smartserve file at the top of `qubit-events.liquid` and `qubit-events-transaction.liquid` with your own unique id.
 
+## Sitewide
 
-##Sitewide
-
-To install the bulk of the Universal Variable code, copy the `uv.liquid` file from this repository to the `Snippets/` directory, found at "Themes > Template Editor > Snippets".
+To install, copy the `qubit-events.liquid` file from this repository to the `Snippets/` directory, found at "Themes > Template Editor > Snippets".
 
 Then at the bottom of the `<head>` in `theme.liquid` in the `Layouts/` directory, add the following code:
 
 ```liquid
-{% include 'uv' %}
+{% include 'qubit-events' %}
 ```
+## Product Variations
 
+If your products have multiple variants (size, color etc) then an additional produce event needs to be emitted when the variant is changed. Due to the way Shopify themes work, it is not possible to template this as each theme can override the switcher. Insert the following event code into the function which handles the variant change:
+```
+  uv.emit('ecProduct', {
+    eventType: 'detail',
+    product: {
+      sku: {{ product.sku | default: product.variant_id | json }},
+      productId: "{{ product.product_id }}",
+      name: {{ product.title | json }},
+      manufacturer: {{ product.vendor | json}},
+      stock: {{ product.inventory_quantity }},
+      price: {
+        currency: "{{ shop.currency }}",
+        value: {{ product.price | divided_by: 100 }}
```

Comment:
Is the standard price format multiplied by 100? We might want to use a [money filter](https://help.shopify.com/themes/liquid/filters/money-filters) here instead.